README.md

```diff
@@ -123,12 +86,10 @@ To use authentication with Streamlit Cloud, please use this TOML format:
 ```toml
 # Environment variables
 OPENAI_API_KEY="sk-xxx"
-ASSISTANT_ID="asst_xxx"
-
-ASSISTANT_TITLE="Assistants API UI"
-ENABLED_FILE_UPLOAD_MESSAGE="Upload a file" # Leave empty to disable
-
-AUTHENTICATION_REQUIRED="True"
+APP_ENABLED_FILE_UPLOAD_MESSAGE="Upload a file" # Leave empty to disable
+# OpenAI assistants
+OPENAI_ASSISTANTS='[{"id": "asst_xxx", "title": "Assistants XXX UI"}, {"id": "asst_yyy", "title": "Assistants YYY UI"}]'
```

Comment:
If you did not set OPENAI_ASSISTANTS, please maintain the traditional UI.

Reply:
> If you did not set OPENAI_ASSISTANTS, please maintain the traditional UI.

have updated to address your review @ryo-ma